package.json

```diff
@@ -102,6 +102,6 @@
   "dependencies": {
     "jstree": "^3.3.3",
     "lodash": "^4.6.1",
-    "moment": "^2.15.2"
+    "moment": "^2.20.1"
   }
 }
```

Comment:
Living life on the edge with an npm update 😄

Reply:
hmm, I think I did that as part of trying to fix Travis when it broke, I'll revert this in favour of what's currently on develop